exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,3 @@
 // Start by creating a variable `greeting`
-
-console.log(greeting);
+var greeting = " hello world";
```

Comment:
A space before the word 'hello' is unnecessary here. I understand that you created it in order to print out 'greeting 'three times in the console including spaces in between. Instead, to fulfil requirements of the task, it would be better to log console three times with 'hello world'.